src/Bulk.php

```diff
@@ -296,6 +297,10 @@ public function send(): ResponseSet
      */
     protected function _processResponse(Response $response): ResponseSet
     {
+        switch ($response->getStatus()) {
+            case 413: throw new RequestEntityTooLargeException();
```

Comment:
Should / can we extract some more from the response? Do we have any message inside that we could add in addition to the exception we throw?

Reply:
Hi, thanks for review.
Unfortunately there is no body in response from ES.
That is why we have this bug.
If there was a message, then there would be no bugs

Reply:
Got it 👍